src/Shared/LogMessagePacketBase.cs

```diff
@@ -800,6 +813,13 @@ private void WriteBuildMessageEventToStream(BuildMessageEventArgs buildMessageEv
             translator.TranslateEnum(ref importance, (int)importance);
         }
 
+        private void WriteResponseFileUsedEventToStream(ResponseFileUsedEventArgs responseFileUsedEventArgs, ITranslator translator)
+        {
+            // code code code code
```

Comment:
👀

Reply:
Oops sorry about that. Left it as a placeholder while trying to solve an issue and forgot to remove it